src/content/en/updates/2017/03/chrome-58-deprecations.md

```diff
@@ -0,0 +1,159 @@
+
+The compatibility risk is low. RFC 2818 has deprecated this for nearly two decades, and the [Baseline Requirements](https://cabforum.org/baseline-requirements-documents/) (which all publicly trusted CAs must abide by) has required the presence of a `subjectAltName` since 2012. Mozilla Firefox already requires the `subjectAltName` for any newly issued publicly trusted certificates since [Firefox 48]( https://bugzilla.mozilla.org/show_bug.cgi?id=1245280 ). 
+
+Note: Enterprises that need to support such certificates for internal purposes may set the `EnableCommonNameFallbackForLocalAnchors` Enterprise policy.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/4v82AueNjaQ/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/4981025180483584) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=308330)
+
+
+## VTTRegion-related bits of TextTrack
+
+The interface elements regions, `addRegion()` and `removeRegion()`, have been removed from the WebVTT spec and will be removed from Chrome in compliance with the [latest spec](https://w3c.github.io/webvtt/). Those needing an alternative can use the `VTTCue.region` property which is being added in Chrome 58.
+
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5308626495340544) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=690014)
+
+
+## WebAudio: remove AudioSourceNode interface
+
+The `AudioSourceNode` interface is not part of the [Web Audio specification](https://www.w3.org/TR/webaudio/), is not constructible, and has no attributes so it basically has no developer-accessible functionality. Therefore it is being removed.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/D-QJm9GCisc/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5753709124386816) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=663818)
+
+
+## Remove webkitdropzone global attribute
+
+The dropzone global attribute was introduced by the [HTML5 drag and drop specification](http://w3c.github.io/html/editing.html#drag-and-drop) as a declarative method for specifying an HTML element's willingness to be the target of a drag-and-drop operation, the content types that can be dropped onto the element, and the drag-and-drop operation (copy/move/link).
+
+Unfortunately, no browser implemented the unprefixed version. Blink and WebKit only implement a prefixed form of the attribute, webkitdropzone. Because the `dropzone` attribute was removed from the spec in [early March 2017](https://github.com/whatwg/html/pull/2402) the prefixed version is being removed from Chrome.
```

Comment:
Can you please remove `Unfortunately` here? I think a more neutral phrase would be "The attribute failed to gain traction among browser vendors."

I don't want to seem like we're mourning the attribute's loss. We (well, I) made it go away 😄